ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/LIVY/role_command_order.json

```diff
@@ -0,0 +1,8 @@
+{
+    "general_deps" : {
+        "_comment" : "dependencies for LIVY",
+        "LIVY_SERVER-START" : ["SPARK_THRIFTSERVER-START","SPARK_JOBHISTORYSERVER-START"],
```

Comment:
The startup of Livy does not seem to rely on the startup of Spark STS; there is no direct relationship between them.

Reply:
This has been resolved with commit [92d736c](https://github.com/apache/ambari/pull/3790/commits/92d736c8bffcf8164d6e6e07dd6759610b85ba0c)